src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -44,18 +44,6 @@ import scala.jdk.CollectionConverters._
 import scala.util.control.NonFatal
 
 object Boot extends IOApp with LazyLogging {
-  val sentryDsn: Option[String] = sys.env.get("SENTRY_DSN")
-  // This value comes from the sbt build as the value of the version key in Version.scala
-  val version: Option[String] = Option(getClass.getPackage.getImplementationVersion)
-  private def initSentry(): Unit = sentryDsn.fold(logger.warn("No SENTRY_DSN found, not initializing Sentry.")) { dsn =>
-    val options = new SentryOptions()
-    options.setDsn(dsn)
-    options.setEnvironment(sys.env.getOrElse("SENTRY_ENVIRONMENT", "unknown"))
-    options.setRelease(version.getOrElse("unknown"))
-
-    Sentry.init(options)
-    logger.info("Sentry initialized")
-  }
```

Comment:
there is a call to this initSentry fn below in boot, just moving it to a companion obj